pages/listings/index.html

```diff
@@ -40,7 +40,7 @@ <h1 class="mt-2 py-1 text-center fw-bold">Job Listings</h1>
             <div class="container d-flex justify-content-center mt-3 col-12 ms-3">
               <div class="col-md-5 w-100 col-10 ms-5 search-container">
                 <div class="input-group">
-                  <label for="searchListing"></label>
+                  <label for="searchListing"><span class="d-none">text</span></label>
 
```

Comment:
Please reduce number of empty lines between codeblocks

Reply:
I would suggest using aria-label here for a more semantic and cleaner look :)

Reply:
dont really understand this, what do you mean

Reply:
By adding the aria-label attribute to the input in question you will have a label without having to add the extra lines of code.

Like this: 
                  <input
                    class="form-control"
                    type="search"
                    placeholder="Search Listings..."
                    id="searchListing"
                    data-search-listings
                    ``aria-label="Search Listings"``
                  />

Reply:
ok

Reply:
The aria-label can go directly in to the input element as an attribute like my example above and the label element can be removed ;)